monasca/templates/_keystone_env.tpl

```diff
@@ -130,4 +141,15 @@ internal Keystone URL and Helm cannot pass more than one variable at once.
   value: "{{ .region_name }}"
 {{- end }}
 {{- end }}
+{{- if .auth_type }}
```

Comment:
is this for v2/v3?

Reply:
v2   python-monascaclient needs to use v2

Reply:
boo